solution-base/mongodb/scripts/set-default-write-concern-majority.sh

```diff
@@ -4,15 +4,24 @@ set_default_majority() {
     mongosh "mongodb://127.0.0.1:27017/?replicaSet=${MONGODB_REPLICA_SET_NAME}"  -u 'root' -p "$MONGODB_ROOT_PASSWORD" <<EOF
 conf = rs.config()
 
-defaultWriteConcern = conf.settings.getLastErrorDefaults
+try {
+    let currentDefault = db.adminCommand({ getDefaultRWConcern: 1 })
 
-if (!defaultWriteConcern || defaultWriteConcern.w !== 'majority') {
-    defaultWriteConcern.w = 'majority'
-    rs.reconfig(conf)
+    if (!currentDefault.defaultWriteConcern || currentDefault.defaultWriteConcern.w !== 'majority') {
+        db.adminCommand({
+            setDefaultRWConcern: 1,
+            defaultWriteConcern: { w: 'majority' }
+        })
+        print("Default write concern set to 'majority'")
+    } else {
+        print("Default write concern already set to 'majority'")
+    }
+} catch (e) {
+    print("Error setting default write concern: " + e)
```

Comment:
error should not be silently ignored : we need to be confident the write concern was set properly...
--> printing the error is fine, but having the script should still fail (and cause set-default-write-concern-majority.sh to fail as well?)

Reply:
please check: if the script did not fail before, then this is probably not a blocker...

but eventually we should make sure Mongo does not start if default RW concern is not correct, or at least we have an alert.